src/main/java/org/wso2/carbon/apimgt/securityenforcer/PingAISecurityHandler.java

```diff
@@ -0,0 +1,328 @@
+/*
+ *  Copyright WSO2 Inc.
```

Comment:
don't we need the year here?

Reply:
fixed with https://github.com/wso2-extensions/apim-handler-pingai/pull/22